tox_packet/src/onion/onion_announce_response.rs

```diff
@@ -140,7 +138,7 @@ pub struct OnionAnnounceResponsePayload {
 impl FromBytes for OnionAnnounceResponsePayload {
     named!(from_bytes<OnionAnnounceResponsePayload>, do_parse!(
         announce_status: call!(AnnounceStatus::from_bytes) >>
-        ping_id_or_pk: map_opt!(take!(32), |bytes: &[u8]| bytes.try_into().ok()) >>
+        ping_id_or_pk: call!(<[u8; 32]>::from_bytes) >>
```

Comment:
Why `32`?

Reply:
Because we don't have appropriate const for this. It can be either sha256 hash or public key. So neither of consts will fit.